chat-components/src/components/notificationpane/NotificationPane.tsx

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import { ILabelStyles, Image, Label, Stack, Link, IStackStyles, IImageStyles, ILinkStyles } from "@fluentui/react";
+import { INotificationPaneInternal } from "./interfaces/common/INotificationPaneInternal";
+import DismissButton from "./subcomponents/DismissButton";
+import CloseChatButton from "./subcomponents/CloseChatButton";
+import { decodeComponentString } from "../../common/decodeComponentString";
+import { Ids } from "../../common/Constants";
+
+function NotificationPane(props: INotificationPaneInternal) {
```

Comment:
Please update ChangeLog at the correct section

Reply:
done